lib/cradle/database/documents.js

```diff
@@ -235,3 +235,40 @@ Database.prototype.remove = function (id, rev) {
 
     remove();
 };
+
+Database.prototype.purge = function(id, rev) {
+    var that = this, doc, args = new(Args)(arguments);
+    function purge() {
```

Comment:
Move the ``purge`` function to the outer scope, and pass the arguments using function parameters.